src/components/Graph/Navigation/Navigation.scss

```diff
@@ -38,12 +38,11 @@
     }
     .btn {
       svg {
-        margin: 0.5em;
         font-size: 0.65em;
-        margin-right: 10px;
+        margin: 0.5em 10px 0.5em 0.5em;
       }
-      padding: 0;
-      padding-right: 0.5em;
+
+      padding: 0 0.5em 0 0;
```

Comment:
how come this choice? possible to have a screenshot?

Reply:
It's just another way of writing css in one-line, it doesnt change anything